src/Client/CollectionClient.php

```diff
@@ -58,20 +58,26 @@ public function list()
         return $this->client->collections->retrieve();
     }
 
-    public function create($name, $fields, $defaultSortingField, array $tokenSeparators, array $symbolsToIndex, bool $enableNestedFields = false)
+    public function create($name, $fields, $defaultSortingField, array $tokenSeparators, array $symbolsToIndex, bool $enableNestedFields = false, array $embed = null)
     {
         if (!$this->client->isOperationnal()) {
             return null;
         }
 
-        $this->client->collections->create([
+        $options = [
             'name'                  => $name,
             'fields'                => $fields,
             'default_sorting_field' => $defaultSortingField,
             'token_separators'      => $tokenSeparators,
             'symbols_to_index'      => $symbolsToIndex,
             'enable_nested_fields'  => $enableNestedFields,
-        ]);
+        ];
+        
+        if ($embed) {
+            $options['embed'] = $embed;
```

Comment:
It looks strange to me this config at this level, are not the `embed` options set at the field level already? I'm probably missing this `embed` options in the official documentation.